releasenotes/notes/dagcircuit-remove-deprecated-bit-methods-8fb2d97ac4abdf4a.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    The deprecated methods :meth:`~qiskit.dagcircuit.DAGCircuit.qubits()` and
+    :meth:`~qiskit.dagcircuit.DAGCircuit.clbits()` have been removed. They are
+    replaced by the :attr:`~qiskit.dagcircuit.DAGCircuit.qubits` and
+    :attr:`~qiskit.dagcircuit.DAGCircuit.clbits` attributes.
```

Comment:
I'd probably also mention the release we deprecated this in here too. Just so people know it's been deprecated for some time.

Reply:
Release note updated in 07e25b5 .